appveyor.yml

```diff
@@ -24,7 +24,6 @@ skip_branch_with_pr: 'true'
 skip_tags: 'true'
 branches:
   only:
-  - develop
```

Comment:
we need to keep 2 lines here, even if develop is not here anymore.
This is because when the yml file is manipulated by the release script, when it has only one entry, the output .yml is not read properly by AppVeyor 😒

Reply:
wtf

Reply:
Yes, it's the .yml -> R list -> .yml process that is not perfect. I will fix it in #470